blocks/bricks/base.py

```diff
@@ -822,7 +822,7 @@ def init(self, *args, **kwargs):
 class ApplicationCall(Annotation):
     """A link between the variable tags and bricks.
 
-    The application call can be used to attach to an apply call auxiliary
+    The application call can be used to attach, to an apply call, auxiliary
```

Comment:
The "to an apply call" seems really obscure to me. Maybe "to attach to the computation graph auxiliary variables..."?